bedrock/careers/templates/careers/home.html

```diff
@@ -104,7 +104,7 @@ <h2 class="mzp-c-card-title">Diversity and Inclusion at Mozilla</h2>
         <div class="mzp-c-card-desc">
           We’ve made some progress that we’re proud of, and we have a lot more work to do.
         </div>
-        <a href="https://blog.mozilla.org/careers/mozilla-diversity-inclusion-2019-results/" class="mzp-c-cta-link">Learn more</a>
+        <a href="https://mozilla.org/diversity/2021/" class="mzp-c-cta-link">Learn more</a>
```

Comment:
D'oh, missed this. Should be:
```suggestion
        <a href="{{ url('mozorg.diversity.2021.index') }}" class="mzp-c-cta-link">Learn more</a>
```